app/README.md

```diff
@@ -22,7 +22,7 @@ Most of the text content in the app can be found within the `template` sections
 
 ## Deployment
 
-To deploy the the app onto either the [production EPA Drupal site](https://wcms.epa.gov) or the [staging app dev server](https://webcms.appdev.epa.gov), verify that the correct paths to the `subregion.json` and `zip.csv` files are both correct in the webpack config file (`prod.config.js` or `stag.config.js`), located in the `webpack` folder.
+To deploy the the app onto either the [production EPA Drupal site](https://beta.epa.gov) or the [staging app dev server](https://webcms.appdev.epa.gov), verify that the correct paths to the `subregion.json` and `zip.csv` files are both correct in the webpack config file (`prod.config.js` or `stag.config.js`), located in the `webpack` folder.
```

Comment:
I think we still keep this as wcms.epa.gov, since they will probably switch from beta to wcms when they cut over. Should we add a section about the beta?